lib/pact/jwt.rb

```diff
@@ -0,0 +1,54 @@
+require 'pact/symbolize_keys'
+module Pact
+
+  # Specifies that the actual object should be considered a match if
+  # it includes the same keys, and the values of the keys are of the same class.
+
+  class Jwt
```

Comment:
What do you think about calling this JWT? It just looks funny to my eye, but I know it's completely subjective.